pkg/kubelet/server/server.go

```diff
@@ -206,7 +215,7 @@ func ListenAndServeKubeletReadOnlyServer(
 	port uint,
 	tp oteltrace.TracerProvider) {
 	klog.InfoS("Starting to listen read-only", "address", address, "port", port)
-	s := NewServer(host, resourceAnalyzer, checkers, nil, nil)
+	s := NewServer(host, resourceAnalyzer, checkers, nil, nil, nil)
```

Comment:
We should have consistency with this and `ListenAndServeKubeletServer` otherwise the 2 kubelet processes running on different ports will show different behaviors

Reply:
Fixed